lighthouse-cli/test/smokehouse/dobetterweb/dbw-expectations.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+/**
+ * Expected Lighthouse audit values for Do Better Web tests.
+ */
+module.exports = [{
+  initialUrl: 'http://localhost:10200/dobetterweb/dbw_tester.html?all',
```

Comment:
do you want to also try the dbw tester against online-only.html to make sure we get passing grades on a fine page?